testsuite/features/step_definitions/cobbler_steps.rb

```diff
@@ -0,0 +1,224 @@
+# Copyright (c) 2023 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+### This file contains the definitions for all steps concerning Cobbler.
+
+# cobbler daemon
+Given(/^cobblerd is running$/) do
+  ct = CobblerTest.new
```

Comment:
As this is just a refactor moving stuff, all good.

But my two cents for the future, this ct = CobblerTest.new in several steps could cause problems when we run tests in parallel, ideally we should have a CobblerAPIHandler global instance, and instantiate it only once. Outside the steps maybe.
But again, out of scope of this PR.